### PR TITLE
Update risk view

### PR DESCRIPTION
**Pull request by sz-piotr**

### CODE REVIEW

File: packages/config/src/projects/optimism.ts [hunk 1]
```diff
@@ -135,14 +130,7 @@ export const optimism: Project = {
...
           ],
-          risks: [
-            // TODO: Uncomment after findings
-            // {
-            //   category: 'Funds can be lost if',
-            //   text: 'the centralized sequencer goes down and nobody can exit.',
-            //   isCritical: true,
-            // },
-          ],
+          risks: [EXITS.RISK_CENTRALIZED_BLOCK_PRODUCER],
```
Comment (sz-piotr) [inline packages/config/src/projects/optimism.ts:133]: Would be great to have a reference here.

File: packages/config/src/projects/common/forceTransactions.ts [hunk 2]
```diff
@@ -35,13 +35,20 @@ const STARKEX_PERPETUAL_WITHDRAW: ProjectTechnologyChoice = {
...
   description:
     WITHDRAW_OR_HALT.description +
-    ' Perpetual positions can also be force closed before withdrawing.',
+    ' Perpetual positions can also be force closed before withdrawing, however this requires the user to find the counterparty for the trade themselves.',
   references: [
     {
       text: 'Censorship Prevention - StarkEx documentation',
       href: 'https://docs.starkware.co/starkex-v3/architecture/overview#8-censorship-prevention',
     },
   ],
```
Comment (sz-piotr) [inline packages/config/src/projects/common/forceTransactions.ts:44]: We need a link about the counterparty here

File: packages/config/src/projects/bobanetwork.ts [hunk 3]
```diff
@@ -107,16 +108,14 @@ export const bobanetwork: Project = {
...
-            'When a user initiates a withdrawal it is processed as a L2 to L1 message. Because Boba Network is an optimistic rollup this transaction has to be included in a block and finalized. This takes several days to happen after which the funds can be withdrawn on L1.',
+          ...EXITS.REGULAR('optimistic', 'merkle proof'),
           references: [
             {
               text: 'The Standard Bridge - Boba documentation',
               href: 'https://docs.boba.network/developer-docs/bridging-l1-l2#the-standardtm-bridge',
             },
           ],
-          risks: [],
+          risks: [EXITS.RISK_CENTRALIZED_BLOCK_PRODUCER],
```
Comment (sz-piotr) [inline packages/config/src/projects/bobanetwork.ts:118]: Would be great to have a reference here.

File: packages/config/src/projects/arbitrum.ts [hunk 4]
```diff
@@ -131,6 +131,7 @@ export const arbitrum: Project = {
               href: 'https://developer.offchainlabs.com/docs/inside_arbitrum#rules-for-confirming-or-rejecting-rollup-blocks',
             },
           ],
+          risks: [EXITS.RISK_CENTRALIZED_BLOCK_PRODUCER],
```
Comment (sz-piotr) [inline packages/config/src/projects/arbitrum.ts:134]: We definitely need a reference here.

File: packages/config/src/projects/common/forceTransactions.ts (hunk 2 above)
Comment (luke7211) [reply packages/config/src/projects/common/forceTransactions.ts:44]: Anyone can publish stateBatch as long as they have a bond. Currently BondManager that checks this is mocked (https://etherscan.io/address/0xCd76de5C57004d47d0216ec7dAbd3c72D8c49057#code) and vouch only for the OVM_Proposer EOA address

File: packages/config/src/projects/optimism.ts (hunk 1 above)
Comment (luke7211) [reply packages/config/src/projects/optimism.ts:133]: Anyone can publish stateBatch as long as they have a bond. Currently BondManager that checks this is mocked (https://etherscan.io/address/0xCd76de5C57004d47d0216ec7dAbd3c72D8c49057#code) and vouch only for the OVM_Proposer EOA address

File: packages/config/src/projects/common/forceTransactions.ts (hunk 2 above)
Comment (luke7211) [reply packages/config/src/projects/common/forceTransactions.ts:44]: https://docs.starkware.co/starkex-v3/starkex-deep-dive/regular-flows/flows-for-off-chain-accounts/forced-operations/perpetual-trading-forced-withdrawal-and-forced-trade#forced-trade

File: packages/config/src/projects/bobanetwork.ts (hunk 3 above)
Comment (luke7211) [reply packages/config/src/projects/bobanetwork.ts:118]: Anyone can publish stateBatch as long as they have a bond. Currently BondManager that checks this is mocked and vouch only for the OVM_Proposer EOA address

File: packages/config/src/projects/optimism.ts (hunk 1 above)
Comment (sz-piotr) [reply packages/config/src/projects/optimism.ts:133]: Hmm we don't have this SC listed. I'll look into it

File: packages/config/src/projects/arbitrum.ts (hunk 4 above)
Comment (luke7211) [reply packages/config/src/projects/arbitrum.ts:134]: https://offchain.medium.com/mainnet-for-everyone-27ce0f67c85e

File: packages/config/src/projects/bobanetwork.ts (hunk 3 above)
Comment (sz-piotr) [reply packages/config/src/projects/bobanetwork.ts:118]: We also don't have a bond manager listed for Boba.

**Pull request merged**